app/javascript/components/Admin/SearchPopover.tsx

```diff
@@ -86,133 +86,137 @@ const SearchPopover = () => {
-          <div className="input">
-            <Icon name="people-fill" />
-            <input
-              name="query"
-              placeholder="Search affiliates (email, name, ID)"
-              type="text"
-              value={data.affiliate_query}
-              onChange={(e) => setData("affiliate_query", e.target.value)}
-            />
-          </div>
-          <Button color="primary" type="submit">
-            <Icon name="solid-search" />
-          </Button>
-        </form>
-
-        <div role="separator">or search by card</div>
-
-        <form
-          onSubmit={(e) => submitForm(e, Routes.admin_search_purchases_path(), null)}
-          style={{ display: "contents" }}
-        >
-          <select name="card_type" value={data.card_type} onChange={(e) => setData("card_type", e.target.value)}>
-            <option value="">Choose card type</option>
-            {card_types.map((cardType) => (
-              <option key={cardType.id} value={cardType.id}>
-                {cardType.name}
-              </option>
-            ))}
-          </select>
-          <div className="input">
-            <Icon name="calendar-all" />
-            <input
-              name="transaction_date"
-              placeholder="Date (02/22/2022)"
-              type="text"
-              value={data.transaction_date}
-              onChange={(e) => setData("transaction_date", e.target.value)}
-            />
-          </div>
-          <div className="input">
-            <Icon name="lock-fill" />
-            <input
-              name="last_4"
-              placeholder="Last 4 (7890)"
-              type="number"
-              value={data.last_4}
-              minLength={4}
-              maxLength={4}
-              onChange={(e) => setData("last_4", e.target.value)}
-            />
-          </div>
-          <div className="input">
-            <Icon name="outline-credit-card" />
-            <input
-              name="expiry_date"
-              placeholder="Expiry (02/22)"
-              type="text"
-              value={data.expiry_date}
-              onChange={(e) => setData("expiry_date", e.target.value)}
-            />
-          </div>
-          <div className="input">
-            <div className="pill">$</div>
-            <input
-              name="price"
-              placeholder="Price (9.99)"
-              type="number"
-              step="0.01"
-              value={data.price}
-              onChange={(e) => setData("price", e.target.value)}
-            />
-          </div>
-          <Button color="primary" type="submit">
-            Search
-          </Button>
-        </form>
-      </div>
+      </PopoverTrigger>
+      <PopoverContent>
+        <div className="grid w-96 max-w-full gap-3">
+          <form
+            onSubmit={(e) => submitForm(e, Routes.admin_search_users_path(), "user_query")}
+            className="input-with-button"
```

Comment:
Please be careful with these - we've removed this class (the one below is fine but this one has been reverted)